relayer/src/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { EventRecord } from "@polkadot/types/interfaces/system";
+import { ParaHeadAndId, ParachainsMap } from "./types";
+import Parachain from "./parachain";
+import Target from "./target";
+import logger from "./logger";
+
+// TODO: consider moving to a separate utils module
+// TODO: implement tests
+export const getParaHeadAndIdFromRecord = ({ event }: EventRecord): ParaHeadAndId => {
```

Comment:
Same comment as in the other place, we probably don't need the whole `EventRecord`, only the field we care about.